Add support for the new "missing" flag to show missing games.

msheehan79 opened the pull request:
Indicators/changes for missing games:
- Prevent Launching a missing game
- Grid view will add a grey overlay to fade it slightly
- If no grid image present, the title text on the grid will be grey instead of white

mmatyas commented on layer_grid/GameGridItem.qml:
```diff
@@ -64,6 +64,16 @@ Item {
+
+        Rectangle {
+            anchors.centerIn: parent
+            visible: game.missing === true ? true : false
```
Let's simplify these bool checks.
```
visible: game.missing
```

msheehan79 replied:
No worries!  Should be all set now.

mmatyas commented on layer_grid/GameGridItem.qml:
```diff
@@ -89,7 +99,7 @@ Item {
         wrapMode: Text.Wrap
         horizontalAlignment: Text.AlignHCenter
-        color: "#eee"
+        color: game.missing === true ? "#808080" : "#eee"
```
`color: game.missing ? "#808080" : "#eee"`

mmatyas commented on theme.qml:
```diff
@@ -167,6 +167,8 @@ FocusScope {
     }
 
     function launchGame() {
+        if (gamegrid.currentGame.missing === true)
```
`if (gamegrid.currentGame.missing)`

mmatyas reviewed (changes requested):
Other than these, it looks good!

msheehan79 committed:
Add support for the new "missing" flag to show missing games.

Indicators/changes for missing games:
- Prevent Launching a missing game
- Grid view will add a grey overlay to fade it slightly
- If no grid image present, the title text on the grid will be grey instead of white

mmatyas commented:
Thank you!
